### ISSUE TITLE
Set Up Rate Limiting

### ISSUE THREAD

**Issue by jmjabour8** [ibm-apiconnect/getting-started#75]
_Suggestion:_ Numbered lists instead of bullet points are really helpful in keeping track of where you are

**Secure your API with a client ID and secret**
I hadn't completed the Client ID and Secret tutorial before, so I had to do that.  Unfortunately I came across a pretty major problem in this tutorial -- even when the client ID and client secret are exactly what they should be, I get the following error message when I try to republish the product:
<img width="835" alt="screen shot 2017-06-28 at 10 33 35 am" src="https://user-images.githubusercontent.com/28990660/27660969-4dc4000a-5c0d-11e7-8062-c230421548ce.png">
It's not the easiest to explain, but essentially no matter what I input as the client ID and client secret value, nothing works.  Even after deleting the client ID and client secret fields altogether within the API, the same error persists.  Feel free to reach out to discuss further.

**Exploring the Default Plan**
(+) Great, no problems here

**Creating a new rate-limited plan**
(+) no problems

**Staging & publishing an updated product to the Sandbox catalog**
(-) Bold "Sandbox" in the third bullet point

**Registering a new (consumer) application**
Suggestion: Highlight or bold "Create a new developer account if necessary, using a different address than your IBM id.)" -- I spent quite a bit of time trying to figure out why I couldn't create a new app, then I realized that I had to log in with separate credentials.  

**Subscribing to an API product**
(+) Great, no problems

**Calling a rate-limited API**
(+) Great, no problems

**Comment by sunitajennings** [comment]
@jmjabour8 I figured out why republish was failing when you went through the tutorial "Secure your API with a client ID and secret". In that tutorial you are instructed to create security definitions for client ID and secret, with the following parameter name values:
X-IBM-Client-ID
X-IBM-Client-Secret

The Client ID parameter name should be X-IBM-Client-Id. Note the lower case d at the end! Once I made that change, repub worked fine.

@lana-li This probably should be addressed in tutorial bluemix/2a. The problem is easily reproducible if you follow the instructions as written.
